### DISCUSSION TITLE
Custom field with leading zero

### DISCUSSION THREAD

**Discussion by IgorRSU** [YetiForceCompany/YetiForceCRM#16093] [General] [answered]
Hi, I have custom field Field type: Text (UiType: 1) Length: 8 Field Mask 99999999
If I`m entering value like 00123456 it's saving as 123456. I tried to create Integer field but result is the same. 
How to keep leading zeroes after saving?

**Comment by rskrzypczak** [comment, marked as answer]
No problem here. The text field keeps the leading zeros. You probably did modify the field types in the database yourself and hence the problem. Create a new Text field from the GUI and you will see that everything works fine. You can verify it on the demos.
